### PR TITLE
Improve examples in README

### PR DESCRIPTION
**Pull request by Tekki**
Changes mentioned in #20.

### CODE REVIEW

**Comment by Tekki** [comment]
The Travis tests are failing. Has nothing to do with the README, but I'll try to fix it.

**Comment by Tekki** [comment]
Still failing. @zakame can you take a look at it?
perl6-docker-travis

Addition: This seems to be a common problem when calling key servers from Docker files. Take a look at the following solution: https://github.com/jacobalberty/unifi-docker/commit/599dff10e60ab13796e97fba21a3ac0b45ddaacd

File: Dockerfile
Comment (tianon) [inline Dockerfile]: This should be just `--batch` on both lines (no need for `--no-tty` with `--batch`).

See https://bugs.debian.org/913614 and all the linked things on https://github.com/docker-library/busybox/pull/55 for more information.

**Comment by tianon** [comment]
Regarding GPG keyserver flakiness, see https://github.com/docker-library/official-images/issues/4252 (especially https://github.com/docker-library/php/pull/666 / https://github.com/tianon/pgp-happy-eyeballs).

File: README.md
Comment (zakame) [inline README.md:35]: Probably note this above as `Simply running a (temporary) container` or somesuch...

File: README.md
Comment (zakame) [inline README.md:41]: ditto

File: Dockerfile
Comment (zakame) [reply Dockerfile]: @tianon could we also just inline the GPG pubkey here (and avoid using keyserver for this altogether?)

Comment (Tekki) [reply Dockerfile]: Changed, and we still have the error.

File: README.md
Comment (Tekki) [reply README.md:35]: Changed.

File: README.md
Comment (Tekki) [reply README.md:41]: And ditto. My idea was that probably most of those who read this don't realize that they create a new container with every `docker run`.

**Comment by tianon** [comment]
> could we also just inline the GPG pubkey here (and avoid using keyserver for this altogether?)

You _could_, but it's kind of heinous for review on our end (and more annoying for users to verify/validate).  See https://gist.github.com/anonymous/362532182df471f1cf72c106cd4427fb / https://gist.github.com/anonymous/306f789c50e2e855e881123b6d83c6d4 for an example of how that ends up looking (which is really, really annoying for users to search to verify that the key being used is the same key published by the project short of downloading and comparing the two files or importing the keys and comparing afterwards).

If you implement just the "happy eyeballs" bit of https://github.com/docker-library/php/pull/666/files#diff-354f30a63fb0907d4ad57269548329e3, this problem should go away for your Travis builds (and they'll then be building in an environment that mimics our official build environment more closely, where we apply very similar DNS hijacking for common PGP keyservers):

```yaml
  - wget -qO- 'https://github.com/tianon/pgp-happy-eyeballs/raw/master/hack-my-builds.sh' | bash
```

**Comment by Tekki** [comment]
"happy eyeballs" added, and I see green lights.  :four_leaf_clover: @tianon thanks for guiding us to the runway!

**Review by tianon — APPROVED** [review]
Anything in particular holding this up?  Anything I (or we, as the official images maintainers) can help out with?

**Review by zakame — APPROVED** [review]
Thanks @Tekki for the work and @tianon for the review! :tada:

**Pull request merged**
